presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorMetadata.java

```diff
@@ -270,6 +274,19 @@ public void testCreateTable()
         assertEquals(columnMetadata.getName(), "orderkey");
         assertEquals(columnMetadata.getType(), BIGINT);
     }
+    @Test
+    public void testCreateTableWithUnsupportedType()
+    {
+        assertNull(metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS));
+
+        try {
+            metadata.createTable(SESSION, getOrdersTableWithUnsupportedType(ImmutableMap.of()), false);
+            assert (false);
```

Comment:
nit: We use `fail()` instead of directly `assert(false)`. It gives you a chance to put in error message. Please check other examples in the same file.

Reply:
Can you change it to using the `fail()` function so it's aligned with other places?

Reply:
I think now there are no more occurrences of this.